Landing page

Rismia opened the pull request.

Rismia committed:
Almost done.

Just need to fix the css a bit because the background image doesnt cover the entire screen everything else is done.

Rismia committed:
I believe im done if you

jmtellez commented:
Should have made the changes on the existing branch so we could follow the previous changes.

jmtellez commented:
When creating a PR make sure to add comments that describe your changes

jmtellez commented on public/index.html:
```diff
@@ -6,19 +6,36 @@
     <!-- Add bootstrap CDN -->
-    <!-- Add Google font of your liking - Example uses Exo Google Font -->
-    <!-- Add stylesheet  -->
+    <style>
+        @import url('https://fonts.googleapis.com/css2?family=Exo:wght@100&display=swap');
+    </style><!-- Add Google font of your liking - Example uses Exo Google Font -->
```
no `<style>` tags on html. Include this font as a <link>

jmtellez replied:
font as a `<link>`

jmtellez commented on public/index.html:
```diff
@@ -6,19 +6,36 @@
-                <!-- Add button with link to Resume - Upload to drive/dropbox -->
+            <div id="home"> 
+                <div class="box">
+                        <h1>Hey, I'm</h1>
+                        <h3>MARIO POIRE</h3>
```
Look at the README images. The names are not the same sizes as the ones you have.

jmtellez commented on public/index.html:
```diff
@@ -6,19 +6,36 @@
+                        <h1>Hey, I'm</h1>
+                        <h3>MARIO POIRE</h3>
+                        <br> <!-- Add Text (h1, h4, p tags )-->
+                        <form action="https://drive.google.com/file/d/1TZuzDv9CkTgCHUwfOh32rSfiqYZZL681/view?usp=sharing">
+                             <button class="button"><span>Resume </span></button>
+                        </form>
```
no need to have a form for this. look into anchor tags `<a href="https://www.w3schools.com">`

jmtellez commented on public/styles/styles.css:
```diff
@@ -0,0 +1,84 @@
+	text-align: center;
+}
+
+h1, .h1 {
+    font-size: calc(1.375rem + 1.5vw);
+    color: white;
+  }
```
h1 is global. if you want to target a specific element add a class to it

jmtellez commented on public/styles/styles.css:
```diff
@@ -0,0 +1,84 @@
+    font-size: calc(1.375rem + 1.5vw);
+    color: white;
+  }
+  @media (min-width: 1200px) {
+    h1, .h1 {
+      font-size: 200%;
+    }
+  }
+
+  h3, .h3 {
+    font-size: calc(1.3rem + 0.6vw);
+    color: yellow;
+  }
+  @media (min-width: 1200px) {
+    h3, .h3 {
+      font-size: 400%;
+    }
+  }
```
what is this doing. Can you explain?

jmtellez reviewed (changes requested):
Look at the comments made and make changes on this branch. dont create a new one

Rismia committed:
Changes.

Fixed the import for the font, and the sizes for the text. Changed the button to an anchor tag and changed the colors of the text with a class.

Rismia committed:
added a comment.

added comment to trigger change.

Rismia committed:
Push

Push

jmtellez reviewed (changes requested):
Font is not being applied. Try troubleshooting it.
Dont need to mess with sizes of `<h1>, <h2>...` 
Add btn class to your button. Look at bootstrap for more reference

Rismia committed:
Additional changes.

Fixed the sizes of the text, switched my button to a btn class, applied fonts to my text.

jmtellez reviewed (approved):
LGTM 👍
